### DISCUSSION TITLE
classes/class_hslider

### DISCUSSION THREAD

**Discussion by giscus** [godotengine/godot-docs-user-notes#228] [User-contributed notes]
# classes/class_hslider

Inherits: Slider< Range< Control< CanvasItem< Node< Object A horizontal slider that goes from left (min) to right (max). Description: A horizontal slider, used to adjust a value by moving a grabber...

https://docs.godotengine.org/en/stable/classes/class_hslider.html

**Comment by GrubGubScratch** [comment]
Can you interact with them on mobile

**Reply by Calinou** [reply]
Yes, as touch events are converted to mouse events by default. There is no native touch event handling in Slider, but it handles mouse events and the touch-to-mouse conversion does a good job at it here.

For better usability on a touchscreen, I suggest using a larger grabber icon as it'll affect how large the grabbable area is.
